.vscode/settings.json

```
@@ -0,0 +1,6 @@
+{
```

By default, let's allow VS Code to format markdown for us.
```suggestion
{
  "editor.defaultFormatter": "esbenp.prettier-vscode",
```

---

Resolved: https://github.com/acmcsufoss/acmOpportunities/pull/1/commits/ab61bc4ad59cbcf21b8df00d42f1b8e9a9bbd854